engines/ospray/OSPRayRenderer.cpp

```diff
@@ -83,6 +83,8 @@ void OSPRayRenderer::commit()
         {
             auto& model =
                 static_cast<OSPRayModel&>(simulationModel->getModel());
+            ospSetObject(_renderer, "secondaryModel",
```

Comment:
move it below where the "world" is set

Reply:
hum... it's specific to simulation...

Reply:
for your plugin maybe, but generically the second model could be used anything